### ISSUE TITLE
Make deterministic ordering possible in coala

### ISSUE THREAD

**Issue by RaitoBezarius** [coala-analyzer/coala#2499]
important/slow

**Comment by Makman2** [comment]
you mean for results?

**Comment by sils1297** [comment]
yeah, results aren't ordered deterministically ATM because if the diffs
differ but everything else doesn't they'll not be ordered

**Comment by Makman2** [comment]
Is this still valid? Imo ordering of results breaks the principle of coala submitting tasks to different processes. If ordering is necessary, a filter should be applied on result-handler side.
